src/rgw/rgw_sync_policy.h

```diff
@@ -342,7 +342,7 @@ struct rgw_sync_pipe_params {
   rgw_user user;
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(2, 1, bl);
+    ENCODE_START(1, 1, bl);
```

Comment:
what happened here?

Reply:
@mattbenjamin it's a new data structure, but I had to modify it through development but didn't want to reconfigure my test environment, so I left it for later.